asset/css/style.css

```diff
@@ -810,6 +919,90 @@ body {
     color: #6070ff;
   }
 
+  /* Modal */
+
+  .popup {
+    position: fixed;
+    background-color: #c1c7d0b0;
+    top: 0;
+    width: 100%;
+    height: 100%;
+    z-index: 101;
+    backdrop-filter: blur(5px);
+    display: none;
+  }
+
+  .modal {
+    position: absolute;
+    background-color: #fff;
+    width: 80%;
+    left: 10%;
+    top: 5%;
+    height: 90vh;
+    border-radius: 16px;
+    overflow-y: scroll;
+    overflow-x: hidden;
+  }
+
+  .modal-header {
+    display: flex;
+    padding: 18px;
+    justify-content: space-between;
+  }
+
+  .modal-close {
+    position: fixed;
+    right: 12%;
+    cursor: pointer;
+  }
+
+  .modal-header h2 {
+    font-weight: 700;
+    font-size: 32px;
+    line-height: 44px;
+    color: #172b4d;
+  }
+
+  .modal-header .frame {
+    padding: 10px 0 0 0;
+  }
+
+  .modal-header .frame .counter1,
+  .modal-header .frame .counter2 {
+    margin: 0 5px;
+  }
+
+  .mobile-img {
+    display: none;
+  }
+
+  .desktop-img {
+    width: 100%;
+    height: 80vh;
+  }
+
+  .modal-body {
+    padding: 0 16px;
+  }
+
+  .modal-footer {
+    display: none;
+  }
+
+  .modal-description {
+    display: flex;
+    padding: 60px 0;
+  }
+
+  .modal-description .modal-buttons {
+    margin: 30px 0;
+    padding: 30px 0;
+    border-top: 1px solid #ebecf0;
+    width: 100%;
+  }
```

Comment:
Great job team 🤝

- [ ] [Optional] You may kindly fix these buttons which seem to appear broken as the screen size reduces on the desktop popup.
![buttons popup](https://user-images.githubusercontent.com/63917725/179850434-8894ce3b-77f2-49c2-8af0-d38ebfa4aa18.PNG)